bench/metrics.mli

```diff
@@ -6,41 +6,41 @@ open Stdune
     [unzip] functions which make serialisation easier. *)
 type ('float, 'int) t =
   { elapsed_time : 'float
-  (** Real time elapsed since the process started and the process
+    (** Real time elapsed since the process started and the process
```

Comment:
I like that the comments beginnings are aligned with the field they're documenting.